packages/react/src/auth/SignInForm.test.tsx

```diff
@@ -649,6 +655,59 @@ describe('SignInForm', () => {
     expect(google.accounts.id.prompt).toHaveBeenCalled();
   });
 
+  test('Google user not found', async () => {
+    const clientId = 'user-not-found';
+    let callback: ((response: GoogleCredentialResponse) => void) | undefined = undefined;
+
+    const google = {
+      accounts: {
+        id: {
+          initialize: jest.fn((args: any) => {
+            callback = args.callback;
+          }),
+          renderButton: jest.fn((parent: HTMLElement) => {
+            const button = document.createElement('div');
+            button.innerHTML = 'Sign in with Google';
+            button.addEventListener('click', () => google.accounts.id.prompt());
+            parent.appendChild(button);
+          }),
+          prompt: jest.fn(() => {
+            if (callback) {
+              callback({
+                clientId,
+                credential: '123123123',
+              });
+            }
+          }),
+        },
+      },
+    };
+
+    (window as any).google = google;
+
+    const onSuccess = jest.fn();
+
+    await act(async () => {
+      await setup({
+        onSuccess,
+        googleClientId: clientId,
+      });
+    });
+
+    await waitFor(() => expect(screen.getByText('Sign in with Google')).toBeInTheDocument());
```

Comment:
```suggestion
    await expect(screen.findByText('Sign in with Google')).resolves.toBeInTheDocument();
```
Maybe worth mentioning that this is the same thing, just in case you're not aware of the `screen.findBy{...}` APIs. They are the async versions of the `getBy{...}` APIs